### ISSUE TITLE
Static resource META-INF not found in native 

### ISSUE THREAD

**Issue by fabry00** [quarkusio/quarkus#17985]
## Describe the bug
META-INF/resources/mytest/docs/index.html not found in native build

### Expected behavior
The request `curl http://localhost:8080/mytest/docs/index.html` should return the file placed in `META-INF/resources/mytest/docs/index.html`  (native build)

### Actual behavior

The command  `curl http://localhost:8080/mytest/docs/index.html`  returns `HTTP/1.1 404 Not Found`

```
* Connected to localhost (::1) port 8080 (#0)
> GET /mytest/docs/index.html HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.75.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 404 Not Found
< content-length: 0
<
* Connection #0 to host localhost left intact
```

## To Reproduce

Have a look at this repo: https://github.com/fabry00/quarkus-static-resources-test/tree/not_working

The `not_working` branch reproduces the issue

- run ./buildDocker.sh
- run ./runDocker.sh
- open a NEW terminal and run ./test.sh

**Comment by geoand** [comment]
I wonder if we should register the `META-INF/resources` directory as a resource directory for native image if `quarkua-vertx-http` is available.

WDYT @cescoffier?

**Comment by cescoffier** [comment]
I'm surprise it's not done. Isn't it the location where we put static assets? How are they served if it's not registered.

**Comment by geoand** [comment]
Good question...

I'd need to have a closer look

**Comment by geoand** [comment]
This has actually already been fixed by https://github.com/quarkusio/quarkus/pull/16989 and the fix is available since `2.0.0.Alpha3`
